src/main/java/Event.java

```diff
@@ -0,0 +1,26 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDateTime from;
+    protected LocalDateTime to;
+    private static final DateTimeFormatter OUTPUT_DATE_FORMAT = DateTimeFormatter.ofPattern("MMM d yyyy, hh:mm a");
+
+    public Event(String description, LocalDateTime from, LocalDateTime to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+    public LocalDateTime getFrom() {
+        return from;
+    }
+
+    public LocalDateTime getTo() {
+        return to;
+    }
```

Comment:
Do you think the method names getFrom and getTo may introduce ambiguity when reading your code in different classes? Perhaps you could change the field names to startTime and endTime and method names to getStartTime and getEndTime?